fdbclient/MultiVersionTransaction.actor.cpp

```diff
@@ -307,6 +321,7 @@ void DLApi::init() {
 	loadClientFunction(&api->transactionReset, lib, fdbCPath, "fdb_transaction_reset");
 	loadClientFunction(&api->transactionCancel, lib, fdbCPath, "fdb_transaction_cancel");
 	loadClientFunction(&api->transactionAddConflictRange, lib, fdbCPath, "fdb_transaction_add_conflict_range");
+	loadClientFunction(&api->transactionGetStorageByteSample, lib, fdbCPath, "fdb_transaction_get_storage_byte_sample");
```

Comment:
In these cases, you'll also need to pass an additional argument to indicate when it is fatal that the function can't be loaded. That would be whenever the header API version is new enough that this function is expected to be present, or 700 in the case of 7.0.

```suggestion
	loadClientFunction(&api->transactionGetStorageByteSample, lib, fdbCPath, "fdb_transaction_get_storage_byte_sample", headerVersion >= 700);
```

Reply:
This is my other question: how and when do I need to increase the API version?